File: src/Services/WareHouse/WareHouse.API/Application/Model/ReportValueTotalDT0.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace WareHouse.API.Application.Model
{
    public class ReportValueTotalDT0 : BaseModel
    {
        public decimal Balance { get; set; }
        public decimal Export { get; set; }
        public decimal Import { get; set; }
        public decimal Beginning { get; set; }
        public string UnitName { get; set; }
        public string WareHouseItemName { get; set; }
        public string WareHouseItemCode { get; set; }

    }
}

File: src/WebClient/WareHouse/src/app/model/ReportValueTotalDT0.ts
import { BaseModel } from "./BaseModel";


export interface ReportValueTotalDT0 extends BaseModel {
    balance: number,
    export: number,
    import: number,
    beginning: number,
    unitName: string,
    wareHouseItemName: string,
    wareHouseItemCode: string,
}